index.bs

```diff
@@ -3242,6 +3287,43 @@ The sample code for generating and registering a new key follows:
       });
 </pre>
 
+## Registration with Platform Authenticator ## {#sample-registration-with-platform-authenticator}
+
+This is flow for when the [=[RP]=] is only interested in creating a public key credential with 
+[=platform authenticators=].
+
+1. The user visits example.com. At this point, the user likely already logged in. 
+
+1. The [=[RP]=] script runs the code snippet below. 
+
+1. The client asks the user if s/he is interested in registering a credenital with platform authenticators. 
+
+1. If the user indicates interest, proceed as described below. Otherwise, stop asking the user for interest. 
+
+1. The authenticator shows appropriate UI and direct user to create a credential. Once the new credential is created, 
+    send it to the server for validation. 
+
+<pre class="example" highlight="js">
+    if (!PublicKeyCredential) { /* Platform not capable of the API. Handle error. */ }
+
+    navigator.credentials.isPlatformAuthenticatorReady()
+        .then(function (userIntent) {
+
+            // If user is interested in the API, start registering credentials.
+            if (userIntent) {
+                var publicKeyOptions = { /* Public key credential creation options. */};
+
+                // Create and register credentials.
+                return navigator.credentials.create({ "publicKey": publicKeyOptions });
+            }
+
+            // If user is not interested, don't ask the user to register. 
```

Comment:
In the sample code, we should put a note that this may never resolve; the catch/then aren't guaranteed to run, etc.